package.json

```diff
@@ -20,12 +20,13 @@
     "node": "^18 || ^20"
   },
   "dependencies": {
-    "@alaskaairux/icons": "^4.38.0",
-    "@aurodesignsystem/auro-dropdown": "^2.10.5",
-    "@aurodesignsystem/auro-input": "^2.17.6",
-    "@aurodesignsystem/auro-library": "^2.5.1",
+    "@alaskaairux/icons": "^4.41.0",
+    "@aurodesignsystem/auro-dropdown": "<=2.10.6",
+    "@aurodesignsystem/auro-formvalidation": "^1.0.3",
+    "@aurodesignsystem/auro-input": "<=2.17.10",
```

Comment:
Why do we need the `<=` here?

Reply:
It was a suggestion from Dale. I believe it was to help eliminate a dependency conflict problem from when people were consuming the complex components like datepicker and combobox, but I don't remember the full details.

Reply:
It won't matter what version the different form elements use and there won't be a conflict because they each register the element with the version number in the tag name.